src/keystore.c

```diff
@@ -385,6 +305,93 @@ static void _delete_retained_seeds(void)
     _retained_bip39_seed_encrypted_len = 0;
 }
 
+keystore_error_t keystore_encrypt_and_store_seed(
```

Comment:
moving these two functions makes it hard to review. Are there any differences, are they the same?